rapids_cli/doctor/checks/gpu.py

```diff
@@ -10,9 +10,10 @@ def gpu_check(verbose=False):
     """Check GPU availability."""
     try:
         pynvml.nvmlInit()
+        num_gpus = pynvml.nvmlDeviceGetCount()
     except pynvml.NVMLError as e:
         raise ValueError("No available GPUs detected") from e
-    num_gpus = pynvml.nvmlDeviceGetCount()
+    assert num_gpus > 0, "No GPUs detected"
```

Comment:
Do we need here the `, "No GPUS detected"`? Under what condition that will, if this message is triggered, isn't it because no GPUs were detected? which already raises in the step above

Reply:
This will raise an assertion error if drivers/nvml is installed but the number of GPUs is zero. The exception above will raise if the NVML calls fail, which is different.